.github/workflows/push-deploy.yml

```diff
@@ -5,7 +5,7 @@ on:
     types: [published]
 
 permissions:
-  contents: read
+  contents: write
   packages: read
```

Comment:
Does this also need to be updated to `write` as done in #345 (and currently underway updating across other plugin repos)?

Reply:
So I know we have this set to `write` in some of our other repos but I don't think it needs to be, though I guess I don't know that for sure and hard to test until we do an actual release. Happy to update that just to be safe if we want

Reply:
I'm going to go ahead and update all the other PRs I opened to remove the `packages: write` change and merge those in so we get everything to the same contents:write/packages:read state and can iterate everything together in the future if needed.